d3a_api_client/redis_aggregator.py

```diff
@@ -173,6 +173,13 @@ def execute_batch_commands(self, is_blocking=True):
                 raise RedisAPIException(f'API registration process timed out.')
         self._client_command_buffer.clear()
 
+    def _on_event_or_response(self, message):
+        logging.info(f"A new message was received. Message information: {message}")
+        log_market_progression(message)
```

Comment:
I think that market progression is not interesting for any other event or response apart from the tick and market cycle, therefore I would recommend to add this only in these 2 events, because as it is now it will clutter up the logs. If the research team wants to log the progression in other events they'll ping us.